integration/spark/README.md

```diff
@@ -91,14 +91,14 @@ The following parameters can be specified in the Spark configuration:
-------------------------------------------|---------------------------------------------------------------------------------------------------|---------------------------------
-| spark.openlineage.transport.type         | The transport type used for event emit, default type is `http`                                    | http                            |
-| spark.openlineage.namespace              | The default namespace to be applied for any jobs submitted                                        | MyNamespace                     |
-| spark.openlineage.parentJobName          | The job name to be used for the parent job facet                                                  | ParentJobName                   |
-| spark.openlineage.parentRunId            | The RunId of the parent job that initiated this Spark job                                         | xxxx-xxxx-xxxx-xxxx             |
-| spark.openlineage.appName                | Custom value overwriting Spark app name in events                                                 | AppName                         |
-| spark.openlineage.facets.disabled        | `;` separated list of facets to disable, default is `spark_unknown;` (currently must contain `;`) | spark_unknown;spark.logicalPlan |
+| Parameter                                | Definition                                                                                                                   | Example                             |
+------------------------------------------|------------------------------------------------------------------------------------------------------------------------------|-------------------------------------
+| spark.openlineage.transport.type         | The transport type used for event emit, default type is `http`                                                               | http                                |
+| spark.openlineage.namespace              | The default namespace to be applied for any jobs submitted                                                                   | MyNamespace                         |
+| spark.openlineage.parentJobName          | The job name to be used for the parent job facet                                                                             | ParentJobName                       |
+| spark.openlineage.parentRunId            | The RunId of the parent job that initiated this Spark job                                                                    | xxxx-xxxx-xxxx-xxxx                 |
+| spark.openlineage.appName                | Custom value overwriting Spark app name in events                                                                            | AppName                             |
+| spark.openlineage.facets.disabled        | List of facets to disable, enclosed in `[]` and separated by `;`, default is `[spark_unknown;]` (currently must contain `;`) | \[spark_unknown;spark.logicalPlan\] |
```

Comment:
It's a pity, we didn't start like that from the beginning. However, we may have now users with disabled facets not contained within `[]` and we don't want their integration to fail after version upgrade. 

I think this PR should be extended, so that: 
 * we clearly inform users that `[]` will be obligatory since 0.21,
 * for the version 0.20 (next one), we should somehow support both approaches (with and without `[]`). Perhaps we can extend the condition to have `if (isArrayType(value) || <<isFacetDisabledNode>>)`

Reply:
Made the changes. Thanks for the review!